includes/wikia/services/RevisionUpvotesService.class.php

```diff
@@ -49,12 +49,14 @@ public function removeUpvote( $id, $fromUser, $userId ) {
 
 		$rowRemoved = $db->affectedRows();
 
-		( new \WikiaSQL() )
-			->UPDATE( self::UPVOTE_USERS_TABLE )
-			->SET_RAW( 'total', 'IF(`total` > 0, `total` - 1, 0)' )
-			->SET_RAW( 'new', 'IF(`new` > 0, `new` - 1, 0)' )
-			->WHERE( 'user_id' )->EQUAL_TO( $userId )
-			->run( $db );
+		if ( $userId > 0 ) {
```

Comment:
I just wonder - would it be better to create an `shouldStoreUpvote()` that could hold more conditions if necessary and would have an explicit naming?